build-logic/src/main/kotlin/buildlogic.chromedriver.gradle.kts

```diff
@@ -18,9 +18,9 @@ repositories {
     exclusiveContent {
         forRepository {
             ivy {
-                url = uri("https://chromedriver.storage.googleapis.com/")
+                url = uri("https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/")
```

Comment:
Is it a trustful source? :)

Reply:
the domain name is registered by Google, see https://www.whois.com/whois/gvt1.com 🤷 
Now trusting Google in general is another topic 😄